stdlib/InteractiveUtils/src/codeview.jl

```diff
@@ -44,17 +44,14 @@ end
 
 """
     code_warntype([io::IO], f, types; debuginfo=:default)
-
```

Comment:
These deletions are accidental I guess?

Reply:
yes, just went through and changed back format